src/routes/exercise-finished/components/ShareSection.tsx

```diff
@@ -12,6 +12,9 @@ const Container = styled.View`
   padding: ${props => props.theme.spacings.sm};
   border-radius: 6px;
   shadow-color: ${props => props.theme.colors.shadow};
+  shadow-offset: 0px 6px;
+  shadowopacity: 0.2;
+  shadowradius: 6;
```

Comment:
`shadowopacity` and `shadowradius` are no valid properties, need to be `shadow-opacity` and `shadow-radius`.